src/main/jhipster/src/main/webapp/app/germplasm-list/list.component.scss

```diff
@@ -0,0 +1,3 @@
+.non-editable {
+  background-color: #a4a4a4;
+}
```

Comment:
we can put it in global (close to the other table classes https://github.com/IntegratedBreedingPlatform/Workbench/blob/73c938be4711262764d3271d7c67fa3838b2f7a3/src/main/jhipster/src/main/webapp/content/scss/global.scss#L277), it'll needed in the future certainly (e.g manage studies).

Reply:
Done